lib/plugins.js

```diff
@@ -274,6 +282,8 @@ function make_delegate (instance, plugin) {
     actmeta.plugin_name = plugin.name || '-'
     actmeta.plugin_tag = plugin.tag || '-'
     actmeta.plugin_fullname = plugin.fullname
+
+    // TODO: is this necessary?
```

Comment:
Hard to prove a negative here. tests pass, but that's not conclusive.
Easy to break other code if there's a dep